src/Promitor.Scraper.Tests.Unit/Serialization/AutoMapperTests.cs

```diff
@@ -0,0 +1,19 @@
+﻿using AutoMapper;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Mapping;
+using Xunit;
+
+namespace Promitor.Scraper.Tests.Unit.Serialization
+{
+    public class AutoMapperTests
+    {
+        [Fact]
+        public void VerifyConfigurationsAreValid()
+        {
+            // Arrange
+            var mapperConfiguration = new MapperConfiguration(c => c.AddProfile<V1MappingProfile>());
+
+            // Act / Assert
+            mapperConfiguration.AssertConfigurationIsValid();
```

Comment:
Will this ensure that everything maps correctly?

Reply:
As things stand I think it will because the model is a one-to-one with the config. So if we renamed the `ResourceGroupName` property on the runtime model to `ResourceGroup` for example, this tests would fail because the property names no-longer match.

In addition, all of the current deserialization tests rely on serializing and then deserializing to the runtime model. As a result of that, they actually test the AutoMapper mappings as well.

Once we've got this PR merged, I'll push up the next stage (altering the runtime model). I've added additional tests for the AutoMapper config to that because there's some parts that need custom mappings to work.

Let me know if that's good enough for you, or whether you want anything else added.